mod-audit-server/src/main/java/org/folio/util/DbUtils.java

```diff
@@ -0,0 +1,11 @@
+package org.folio.util;
+
+import static java.lang.String.format;
+import static org.folio.rest.persist.PostgresClient.convertToPsqlStandard;
+
+public class DbUtils {
+
```

Comment:
please add private constructor for utility classes

Reply:
added